src/main/java/seedu/address/model/lab/Lab.java

```diff
@@ -15,12 +15,12 @@
 public class Lab {
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Lab number should be a valid positive integer";
+            "Lab number should be an integer between 0 and 20 inclusive.";
 
     /*
      * Lab number has to be a positive Integer.
      */
-    public static final String VALIDATION_REGEX = "[1-9]\\d*";
+    public static final String VALIDATION_REGEX = "[0-9]\\d*";
```

Comment:
Hmm I think this can be simplified to `\\d*` (but same effect)

Reply:
Oh yes thanks for the suggestions